Dockerfile-debug

```diff
@@ -1,19 +1,15 @@
-FROM debian:bookworm-slim
-RUN apt-get update && apt-get install -y --no-install-recommends \
+FROM alpine:3.20.1
+RUN apk add \
+    bind-tools \
     curl \
-    dnsutils \
     iptables \
+    iptables-legacy \
     jq \
     nghttp2 \
     tcpdump \
     iproute2 \
     lsof \
-    conntrack \
-    tshark && \
-    rm -rf /var/lib/apt/lists/*
-
-# We still rely on old iptables-legacy syntax.
-RUN update-alternatives --set iptables /usr/sbin/iptables-legacy \
-    && update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
```

Comment:
I removed this, since alpine doesn't have `update-alternatives`. On the old debug image you'd run `iptables` for legacy and `iptables-nft` for nft. On the new image you'd instead run `iptables-legacy` for legacy and `iptables` for nft. Maybe that's ok?